async_grpc/BUILD.bazel

```diff
@@ -61,11 +61,14 @@ cc_library(
     ]),
     copts = ["-Wno-sign-compare"],
     includes = ["."],
+    defines = ["TRACING_SUPPORT"],
```

Comment:
Cartographer style: rename to BUILD_TRACING(_SUPPORT)
maybe copts is enough unless this needs to be exported